src/schema/auction_result.js

```diff
@@ -106,6 +109,35 @@ const AuctionResultType = new GraphQLObjectType({
     low_estimate: amount(({ low_estimate_cents_usd }) => low_estimate_cents_usd),
     high_estimate: amount(({ high_estimate_cents_usd }) => high_estimate_cents_usd),
     price_realized: amount(({ price_realized_cents_usd }) => price_realized_cents_usd),
+
+    prices: {
+      type: new GraphQLObjectType({
+        name: "AuctionResultPrices",
+        fields: {
+          display: {
+            type: GraphQLString,
+          },
+        },
+      }),
+      resolve: ({ price_realized_cents, currency }) => {
+        const { symbol, subunit_to_unit } = currencyCodes[currency.toLowerCase()]
+        let display = ""
+
+        if (symbol) {
+          display = symbol
+        }
+
+        if (indexOf(symbolOnly, currency) === -1) {
+          display += " " + currency
+        }
+
+        display += Math.round(price_realized_cents / subunit_to_unit).toLocaleString()
+
+        return {
+          display,
+        }
```

Comment:
Rather than returning an object with all (future) fields pre-computed, return an empty object here and move the field computation up into the `display` field’s resolver so it gets computed on-demand.

Reply:
Ah yea, the tricky `resolve: () => ({})`

That's a better structure, for sure. 👍

Reply:
Actually you do have to just resolve with the auction lot itself, rather than the tricky empty object (since we need some fields from the lot object in the resolvers of the fields of `prices`).

Reply:
Ah yeah indeed 👍